packages/ui-library/src/icons/emptyWallet.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { CustomSvgIcon, SvgIconProps } from '../components/svgicon';
+
+interface IconProps extends SvgIconProps {
+  size?: string;
+}
+
+export default function EmptyWalletIcon({ size, ...props }: IconProps) {
+  return (
+    <CustomSvgIcon viewBox="0 0 20 20" style={size ? { fontSize: size } : {}} {...props}>
+      <path
+        d="M14.784 18.9585H6.35898C6.09232 18.9585 5.85898 18.7918 5.76732 18.5501C5.67565 18.3001 5.75065 18.0251 5.95065 17.8584C6.15065 17.6918 6.33399 17.4751 6.46732 17.2418C6.73399 16.8168 6.86732 16.3335 6.86732 15.8418C6.86732 14.3501 5.65065 13.1335 4.15898 13.1335C3.54232 13.1335 2.95898 13.3418 2.46732 13.7334C2.27565 13.8834 2.02565 13.9085 1.80898 13.8085C1.59232 13.7085 1.45898 13.4835 1.45898 13.2418V9.60014C1.45898 7.52514 3.14232 5.8418 5.21732 5.8418H14.784C16.859 5.8418 18.5423 7.52514 18.5423 9.60014V10.8001C18.5423 11.1418 18.259 11.4251 17.9173 11.4251H16.234C15.9423 11.4251 15.6757 11.5335 15.484 11.7335L15.4757 11.7418C15.2423 11.9668 15.134 12.2751 15.159 12.5918C15.209 13.1418 15.734 13.5834 16.334 13.5834H17.9173C18.259 13.5834 18.5423 13.8668 18.5423 14.2084V15.2001C18.5423 17.2751 16.859 18.9585 14.784 18.9585ZM7.65065 17.7085H14.784C16.1673 17.7085 17.2923 16.5834 17.2923 15.2001V14.8334H16.334C15.0757 14.8334 14.009 13.9001 13.909 12.7001C13.8423 12.0168 14.0923 11.3418 14.5923 10.8501C15.0257 10.4085 15.609 10.1668 16.234 10.1668H17.2923V9.5918C17.2923 8.20846 16.1673 7.08346 14.784 7.08346H5.21732C3.83399 7.08346 2.70898 8.20846 2.70898 9.5918V12.1585C3.17565 11.9751 3.66732 11.8751 4.16732 11.8751C6.35065 11.8751 8.12565 13.6501 8.12565 15.8335C8.12565 16.4918 7.95898 17.1418 7.65065 17.7085Z"
+        fill="#06AC54"
```

Comment:
remember to remove the `fills` on all the new icons! if not they are always going to have the same color :)

Reply:
Good catch! Thank you